BoxContentSDK/BoxContentSDK/BOXContentSDKConstants.m

```diff
@@ -467,3 +467,9 @@
 NSString *const BOXURLSessionTaskCacheResponseData = @"responseData";
 NSString *const BOXURLSessionTaskCacheError = @"error";
 NSString *const BOXURLSessionTaskCacheUserIdAndAssociateId = @"userIdAndAssociateId";
+
+const BOXRepresentationHints BOXRepresentationHintsDefaultThumbnails = @"[jpg?dimensions=320x320][jpg?dimensions=1024x1024&paged=false]";
+// We separate HLS here because HLS is only suitable for streaming, and there will be cases
+// we want to fall back on MP4
+const BOXRepresentationHints BOXRepresentationHintsDefaultPreview = @"[hls][pdf,mp4,mp3,jpg?paged=false]";
+const BOXRepresentationHints BOXRepresentationHintsDefaultThumbnailsandPreview = @"[jpg?dimensions=320x320][jpg?dimensions=1024x1024&paged=false][hls][pdf,mp4,mp3,jpg?paged=false]";
```

Comment:
Are you sure that `jpg?paged=false` is what you want? Have you tested this alone and found that it returns the asset you want?

Reply:
Our preview SDK (with which this is generally used) doesn't yet handle paged reps, so this seems to be the right call for now. I'll test, but I'd expect this probably to get an unpaged jpg 2048 if available.

Reply:
Ah, that rep doesn't exist, hence why I suggested the testing.